app/MMB/src/scenes/passenger/Passenger.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation, DateFormatter } from '@kiwicom/mobile-localization';
+import { SimpleCard, StyleSheet, Text, Color } from '@kiwicom/mobile-shared';
+import idx from 'idx';
+import { View } from 'react-native';
+import startCase from 'lodash/startCase';
+
+import { SeparatorFullWidth } from '../../components/Separators';
+import type { Passenger_passenger as PassengerType } from './__generated__/Passenger_passenger.graphql';
+
+type RowItemProps = {|
+  title: React.Element<typeof Translation>,
+  value: React.Element<typeof Translation>,
+|};
+
+const RowItem = (props: RowItemProps) => (
+  <View style={styles.rowItem}>
+    <Text style={styles.titleLight}>{props.title}</Text>
+    {props.value}
+  </View>
+);
+
+type Props = {|
+  passenger: PassengerType,
+|};
+
+export class Passenger extends React.Component<Props> {
+  render = () => {
+    const { passenger } = this.props;
+    if (passenger == null) {
+      return null;
+    }
+    const title = idx(passenger, _ => _.title) || '';
+    const firstname = idx(passenger, _ => _.firstname) || '';
+    const lastname = idx(passenger, _ => _.lastname) || '';
+    const birthday = idx(passenger, _ => _.birthday) || '';
+    const nationality = idx(passenger, _ => _.nationality) || '';
+    const idNumber = idx(passenger, _ => _.travelDocument.idNumber) || '';
+    const insuranceType = startCase(
+      (idx(passenger, _ => _.insuranceType) || '').toLowerCase(),
+    );
+    return (
+      <SimpleCard style={StyleSheet.flatten(styles.card)}>
+        <Text style={styles.name}>
+          <Translation passThrough={`${title}. ${firstname} ${lastname}`} />
```

Comment:
What about leaving this to our GraphQL? Field `fullName` would be better since this can be quite complicated logic and there is going to be more places where we need the full name...

Reply:
sounds doable